### PR TITLE
Resolve angular-bootstrap conflict

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: bower.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "angular": "~1.3.0",
     "angular-bootstrap": "~0.11.1",
     "angular-route": "~1.3.0",
```
Comment (stulees) [inline bower.json]: @donnapep I think instead of adding the resolution you can change the version here to `^0.13.0` like we do in rss widget

Comment (donnapep) [inline bower.json]: @stulees I'd tried that, but it still stops to ask for a resolution.

`Unable to find a suitable version for angular-bootstrap, please choose one:
    1) angular-bootstrap#~0.11.1 which resolved to 0.11.2 and is required by widget-settings-ui-components#1.2.12
    2) angular-bootstrap#^0.13.0 which resolved to 0.13.4 and is required by component-storage-selector#1.1.2, component-subscription-status#0.4.0, widget-text`

I'd probably have to update those other dependencies, but I don't want to risk breaking anything.

Comment (stulees) [inline bower.json]: @donnapep Oh I see, it's cause of the older version of widget-settings-ui-components being targeted. Thanks

**Comment by stulees** [comment]
@donnapep :+1:

File: bower.json (hunk 1 above)
Comment (donnapep) [inline bower.json]: @stulees You got it. Thx.

**Pull request merged**